### ISSUE TITLE
bigip_do showing json difference even when there is not any

### ISSUE THREAD

**Issue by mleklund** [F5Networks/terraform-provider-bigip#557]
### Environment

 * TMOS/Bigip Version:
 * Terraform Version:
 * Terraform bigip provider Version:

### Summary

Please also include information about the reproducibility and the severity/impact of the issue.

### Steps To Reproduce

```
resource "bigip_do"  "do-lbltest" {
  do_json = "${file("lbltest.json")}"
  depends_on = [null_resource.install_do_lbltest]
}
```

```
{
  "schemaVersion": "1.24.0",
  "class": "Device",
  "async": true,
  "label": "my BIG-IP declaration for declarative onboarding",
  "Common": {
    "class": "Tenant",
    "mySystem": {
      "class": "System",
      "hostname": "lbltest",
      "cliInactivityTimeout": 1200,
      "consoleInactivityTimeout": 1200,
      "autoPhonehome": false,
      "preserveOrigDhcpRoutes": false
    },
    "deviceCertificate": {
      "class": "DeviceCertificate",
        "certificate": {
          "base64": "XXXXXXXXXXXXXX"
        },
        "privateKey": {
          "base64": "XXXXXXXXXXXXXX"
        }
    },
    "root": {
      "class": "User",
      "userType": "root",
      "oldPassword": "XXXX",
      "newPassword": "XXXX"
    },
    "admin": {
      "class": "User",
      "userType": "regular",
      "password": "XXXX",
      "shell": "bash"
    }
  }
}
```

### Expected Behavior
The terraform saved state to match our json input after apply.

### Actual Behavior

It appears the terraform state is saving the json returned by the DO post, which is sanitized of passwords and ssl keys, making the configuration appear to have drift.  In a full terrafrom provider we could use a lifecycle ignore_changes block, but we cannot do this with a encoded json block.

```
  ~ resource "bigip_do" "do-lbltest" {
      ~ do_json = jsonencode(
          ~ {
              ~ Common        = {
                  ~ admin             = {
                      + password = "XXXXXX"
                        # (3 unchanged elements hidden)
                    }
                  ~ deviceCertificate = {
                      + privateKey  = {
                          + base64 = "XXXXXXXXXXXXXXXX"
                        }
                        # (2 unchanged elements hidden)
                    }
                  ~ root              = {
                      + newPassword = "XXXXXX"
                      + oldPassword = "XXXXXX"
                        # (2 unchanged elements hidden)
                    }
                    # (17 unchanged elements hidden)
                }
                # (4 unchanged elements hidden)
            }
        )
        id      = "91ade445-1340-4fac-8db6-424eab11c0bb"
        # (1 unchanged attribute hidden)
    }
```

**Comment by trinaths** [comment]
Created INFRAANO-556 for internal tracking.

**Comment by jbaribeault** [comment]
Any updates on this? I'm experiencing the same issue - the stripped fields make it appear there are changes and there are no ignore_changes hooks on a jsonencoded string. DO 1.24.0

In fact - many other fields (from standard/default values on the F5 side - think "all-properties" from tmsh) are showing up as needing to be changed to null.

```
~ VLAN_40             = {
                      - autoLastHop     = "default" -> null
                      - cmpHash         = "default" -> null
                      - failsafeAction  = "failover-restart-tm" -> null
                      - failsafeEnabled = false -> null
                      - failsafeTimeout = 90 -> null
                      - mtu             = 1500 -> null
                        # (3 unchanged elements hidden)
```
Happens for VLANs, routes, failover addresses, mirror addresses, SNMP community strings to name a few.

**Comment by nmenant** [comment]
we are investigating options.  As soon as we have a plan, we will share the proposal with you

**Comment by mleklund** [comment]
yes, any places where the do json does not match what is returned from the do post will show up as differences.  Makes it near impossible to add to a ci/cd system.

**Comment by RavinderReddyF5** [comment]
@mleklund fixed in v1.12.1
